optax/_src/transform_test.py

```diff
@@ -202,6 +202,35 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  @parameterized.parameters(
+    (0.01, 0.55, 123),
+    (0.3, 0.54, 234),
+    (1.0, 0.56, 314),
+  )
+  def test_add_noise(self, eta, gamma, seed):
+    # Prepare to compare noise with a rescaled unit-variance substitute.
+    noise = transform.add_noise(eta, gamma, seed)
+    noise_unit = transform.add_noise(1.0, 0.0, seed)
+
+    params = self.init_params
+    state = noise.init(params)
+    state_unit = noise_unit.init(params)
+
+    # Check the noise itself by adding it to zeros.
+    updates = jax.tree_map(jnp.zeros_like, self.per_step_updates)
+
+    for i in range(1, STEPS + 1):
+      updates_i, state = noise.update(updates, state)
+      updates_i_unit, state_unit = noise_unit.update(updates, state_unit)
+
+      scale = jnp.sqrt(eta / i**gamma)
+
+      def rescale(g):
+        return g * scale  # pylint:disable=cell-var-from-loop
```

Comment:
I think this is simple enough to be written as lambda in the line below. That way you also wouldn't need to disable the pylint warning or did pylint complain about something else then?

Reply:
Yes this is because pylint complains with the lambda form, and with this same warning. This style copies [L186](https://github.com/deepmind/optax/blob/a0d5ed5539a58cc0bfa1d5948c87d2aa3b693d92/optax/_src/transform_test.py#L186).

Reply:
Neat solution, done with the lambda.